osbenchmark/worker_coordinator/worker_coordinator.py

```diff
@@ -2509,6 +2522,137 @@ async def __call__(self, *args, **kwargs):
                 self.complete.set()
             await self._cleanup()
 
+class UnhingedExecutor:
```

Comment:
Nit: Same here, can we change this to `AsyncNoAwaitExecutor` or something similar to maintain naming convention?